sdk/endpoints.py

```diff
@@ -15,8 +15,8 @@
 from dataclasses import dataclass
 from typing import Final
 
-CORE_API_URL: Final[str] = "https://apis.dev.pasqal.cloud/core"
-ACCOUNT_API_URL: Final[str] = "https://apis.dev.pasqal.cloud/account"
+CORE_API_URL: Final[str] = "http://localhost:4200"
```

Comment:
Is it an equivalent ? if not what triggered this change ?

Reply:
No it's an oversight, this sould have not been commited I'll revert it.